modules/gsl/metadata.json

```diff
@@ -0,0 +1,5 @@
+{
+  "versions": [
```

Comment:
Hmm, just noticed the information in metadata.json isn't complete. Can you add missing fields?

for example, https://github.com/bazelbuild/bazel-central-registry/blob/main/modules/abseil-cpp/metadata.json

I'm a bit embarrassed that we don't have validation for this.